Documentation/Testing/ExtensionTesting.rst

```diff
@@ -364,59 +364,66 @@ Rather than changing the :file:`runTests.sh` to then use `greadlink` and thus ri
+           php: [ '7.2', '7.3', '7.4' ]
+           minMax: [ 'composerInstallMin', 'composerInstallMax' ]
+       steps:
+         - name: Checkout
+           uses: actions/checkout@v2
 
-    before_script:
-      - Build/Scripts/runTests.sh -s composerInstall -p $TRAVIS_PHP_VERSION
+         - name: Composer
+           run: Build/Scripts/runTests.sh -p ${{ matrix.php }} -s ${{ matrix.minMax }}
 
-    script:
-      - >
-        echo "Running composer validate"
-        Build/Scripts/runTests.sh -s composerValidate -p $TRAVIS_PHP_VERSION
-      - >
-        echo "Running unit tests";
-        Build/Scripts/runTests.sh -s unit -p $TRAVIS_PHP_VERSION
-      - >
-        echo "Running php lint";
-        Build/Scripts/runTests.sh -s lint -p $TRAVIS_PHP_VERSION
+         - name: Composer validate
+           run: Build/Scripts/runTests.sh -p ${{ matrix.php }} -s composerValidate
 
-In case of enetcache, we let Travis CI test the extension with the two PHP versions 7.2 and 7.3. Travis exposes
-the current version as the variable `$TRAVIS_PHP_VERSION`, so we use that to feed it to `runTests.sh`. We instruct
-Travis to always `composer install` first, then run the test suites `composer validate`, the unit testing
-and the PHP linting. It's possible to see executed test runs `online <https://travis-ci.org/lolli42/enetcache>`_.
-Green :) Maybe it's now time to add the `travis-ci status badge <https://docs.travis-ci.com/user/status-images/>`_
-to our README.md file.
+         - name: Lint PHP
+           run: Build/Scripts/runTests.sh -p ${{ matrix.php }} -s lint
 
-Note we again use :file:`runTests.sh` to actually run tests. So the environment our tests are executed in is
-identical to our local environment. It's all dockerized. We don't care about the PHP versions travis-ci loaded
-and installed for us. Travis CI needs the setting `sudo: true` to allow starting own containers, though.
+         - name: Unit tests
+           run: Build/Scripts/runTests.sh -p ${{ matrix.php }} -s unit
+
+         - name: Functional tests with mariadb
+           run: Build/Scripts/runTests.sh -p ${{ matrix.php }} -d mariadb -s functional
 
-Travis CI comes with many additional options and possibilities. If for instance we want to run multiple jobs
-in parallel check out `Travis' Build Stages feature <https://docs.travis-ci.com/user/build-stages/>`_.
+         - name: Functional tests with mssql
+           run: Build/Scripts/runTests.sh -p ${{ matrix.php }} -d mssql -s functional
+
+         - name: Functional tests with postgres
+           run: Build/Scripts/runTests.sh -p ${{ matrix.php }} -d postgres -s functional
+
+         - name: Functional tests with sqlite
+           run: Build/Scripts/runTests.sh -p ${{ matrix.php }} -d sqlite -s functional
+
+In case of enetcache, we let Github Actions test the extension with the two PHP versions 7.2, 7.3 and 7.4.
+All defined steps run on the same checkout, so we will see three test runs in total, one per PHP version.
```

Comment:
Actually we will have 6 jobs running in parallel due to the matrix and the newly added Composer min/max suite.